staging/src/k8s.io/apiserver/pkg/endpoints/filters/traces.go

```diff
@@ -32,6 +33,14 @@ func WithTracing(handler http.Handler, tp trace.TracerProvider) http.Handler {
 		otelhttp.WithPropagators(tracing.Propagators()),
 		otelhttp.WithPublicEndpoint(),
 		otelhttp.WithTracerProvider(tp),
+		otelhttp.WithSpanNameFormatter(func(operation string, r *http.Request) string {
+			ctx := r.Context()
+			info, exist := request.RequestInfoFrom(ctx)
+			if !exist {
+				return "KubernetesAPI"
```

Comment:
From https://opentelemetry.io/docs/specs/semconv/http/http-spans/#name:

> If there is no (low-cardinality) http.route available, HTTP server span names SHOULD be [{method}](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#method-placeholder).

```suggestion
				return r.Method
```